dindex_store/fulltext_index_duckdb.py

```diff
@@ -82,13 +69,11 @@ def insert(self, profile_id, dbName, path, sourceName, columnName, data) -> bool
     # Query Methods
 
     def fts_query(self, keyword, search_domain, max_results, exact_search) -> List:
-        # TODO: switch between exact/approx search ("exact_search")
-
         # FIXME: translate search_domain into a field; here defaulting to data
         search_domain = 'data'
 
         query = f"""WITH scored_docs AS (
-                SELECT *, fts_main_{self.table_name}.match_bm25(data, '{keyword}', fields := '{search_domain}', conjunctive := 1) 
+                SELECT *, fts_main_{self.table_name}.match_bm25(data, '{keyword}', fields := '{search_domain}', conjunctive := {1 if exact_search else 0}) 
```

Comment:
I'm curious about this line. Will this change allow fuzzy keyword search?

Reply:
Yes, that's the purpose!